packages/app/package.json

```diff
@@ -1,217 +1,210 @@
 {
```

Comment:
Could we get this file to two spaces as spacing again, that will make the diff easier to read 😄

Reply:
Ah yeah, it is prettier messing up... how do I make prettier only save on change if there is a prettier config in the project? You know? :)

Reply:
So weird... my editor refuses to format it correctly... anyways, fixed it :)